package proto_ff;

import "yd_fieldoptions.proto";


message dragonWorldWarconstants
{
	optional int32 ID = 1[(yd_fieldoptions.field_cname) = "编号"];
	optional int32 valuedata = 2[(yd_fieldoptions.field_cname) = "值数据"];
	optional string stringdata = 3[(yd_fieldoptions.field_cname) = "文本数据", (yd_fieldoptions.field_bufsize) = 117];
}


message Sheet_dragonWorldWarconstants
{
	repeated dragonWorldWarconstants dragonWorldWarconstants_List = 1[(yd_fieldoptions.field_arysize)=119];
}

message dragonWorldWargoldskill
{
	optional int32 ID = 1[(yd_fieldoptions.field_cname) = "编号"];
	optional int32 type = 2[(yd_fieldoptions.field_cname) = "加成类型"];
	optional int32 goldmin = 3[(yd_fieldoptions.field_cname) = "阶段底限"];
	optional int64 skillId = 4[(yd_fieldoptions.field_cname) = "技能ID"];
}


message Sheet_dragonWorldWargoldskill
{
	repeated dragonWorldWargoldskill dragonWorldWargoldskill_List = 1[(yd_fieldoptions.field_arysize)=119];
}

message dragonWorldWargoldShop
{
	optional int32 ID = 1[(yd_fieldoptions.field_cname) = "编号"];
	optional int64 itemId = 2[(yd_fieldoptions.field_cname) = "道具ID"];
	optional int32 itemNum = 3[(yd_fieldoptions.field_cname) = "道具数量"];
	optional int32 itemPrice = 4[(yd_fieldoptions.field_cname) = "道具购买积分"];
	optional int32 weight = 5[(yd_fieldoptions.field_cname) = "权重"];
}


message Sheet_dragonWorldWargoldShop
{
	repeated dragonWorldWargoldShop dragonWorldWargoldShop_List = 1[(yd_fieldoptions.field_arysize)=139];
}
